app/views/layouts/base.html.erb

```diff
@@ -61,7 +61,9 @@ See docs/COPYRIGHT.rdoc for more details.
         data-first-day-of-week="<%= locale_first_day_of_week %>"
         data-default-locale="<%= I18n.default_locale %>"
         data-environment="<%= Rails.env %>"
-        data-edition="<%= OpenProject::Configuration.edition %>"/>
+        data-edition="<%= OpenProject::Configuration.edition %>"
+        data-attachments-storage="<%= OpenProject::Configuration.remote_storage? ? 'remote' : 'local' %>"
```

Comment:
Why did you need to add this? As far as I can see, checking for the `prepareAttachments` link is enough to find out whether direct uploads are enabled?

Reply:
I agree but this is in case people do not want direct uploads for what ever reason even if it's possible. Jens mentioned that this may be useful in certain installations.